src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Services/SyntaxFacts/CSharpSyntaxFacts.cs

```diff
@@ -1181,6 +1181,25 @@ public bool IsInferredAnonymousObjectMemberDeclarator([NotNullWhen(true)] Syntax
         => node is AnonymousObjectMemberDeclaratorSyntax anonObject &&
            anonObject.NameEquals == null;
 
+    public bool IsInferredTupleMemberDeclarator([NotNullWhen(true)] SyntaxNode? node)
+        => node is not null
```

Comment:
Do not mix expressions with ands and ors. Just make a normal method with statements